lib/Doctrine/ORM/Configuration.php

```diff
@@ -164,7 +164,7 @@ public function setMetadataDriverImpl(MappingDriver $driverImpl)
      *
      * @return AnnotationDriver
      */
-    public function newDefaultAnnotationDriver($paths = [], $useSimpleAnnotationReader = true)
+    public function newDefaultAnnotationDriver($paths = [], $useSimpleAnnotationReader = true, bool $reportFieldsWhereDeclared = false)
```

Comment:
Just to clarify, the plan is to deprecate this new argument in ORM 3.1?

Reply:
Yes, or in 3.0? Not so sure what the release policy mandates/allows.

To my understanding, in Symfony the „next .0“ equals „the last .x minus the compat layer“, so there are no additional features or deprecations. So, there is no room for new deprecations which will go into the next .1 release.

But I don’t mind, if you think 3.0 makes more sense that’s fine for me.

Reply:
3.1 sounds good to me 👍